gitmessage

```diff
@@ -1,11 +1,20 @@
+# 50-character subject line
 
+# 72-character wrapped description.
 
-# 50-character subject line
-#
-# 72-character wrapped longer description. This should answer:
-#
-# * Why was this change necessary?
-# * How does it address the problem?
-# * Are there any side effects?
-#
-# Include a link to the ticket, if any.
+Why this is necessary:
```

Comment:
Do you intend for users to actually include this text in the commit message, or do you expect people to rewrite or remove this?